sympy/physics/continuum_mechanics/beam.py

```diff
@@ -901,8 +901,8 @@ def max_shear_force(self):
                 val = []
                 for point in points:
                     val.append(shear_curve.subs(x, point))
-                points.extend([singularity[i-1], s])
-                val.extend([limit(shear_curve, x, singularity[i-1], '+'), limit(shear_curve, x, s, '-')])
+                points.extend((singularity[i-1], s))
+                val.extend((limit(shear_curve, x, singularity[i-1], '+'), limit(shear_curve, x, s, '-')))
                 val = list(map(abs, val))
                 max_shear = max(val)
```

Comment:
Maybe
```python
val += [
    abs(limit(shear_curve, x, singularity[i-1], '+')),
    abs(limit(shear_curve, x, s, '-'))),
]
```
I guess this isn't tested with symbolic inputs because `max(val)` would probably fail in that case.

Reply:
Yes, plus `val.append(abs(shear_curve.subs(x, point)))` on the lines above.